index.qmd

```diff
@@ -11,3 +11,7 @@ date: "`r Sys.Date()`"
 A catalog of Tables, Listings, and Graphs output produced by NEST tools.
 
 **See the full index of available TLG templates on the [Index page](tlg-index.qmd).**
+
+## License
+
+This book as well as code examples are licensed under the Apache License, Version 2.0.
```

Comment:
book?

Reply:
Do you think that "website" is more suitable?

Reply:
catalog or repository maybe?

Reply:
catalog sounds better

Reply:
catalog i think is good.

Reply:
ok - will open PR